.github/workflows/coverage.yaml

```diff
@@ -40,7 +41,8 @@ jobs:
         if: github.actor != 'dependabot[bot]'
         env:
           COVERALLS_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           MAVEN_OPTS: -Dorg.slf4j.simpleLogger.defaultLogLevel=warn -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
         run: |
           mvn -v
-          mvn clean jacoco:prepare-agent test jacoco:report coveralls:report -DrepoToken=$COVERALLS_TOKEN  --batch-mode
+          mvn clean jacoco:prepare-agent test jacoco:report coveralls:report org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -DrepoToken=$COVERALLS_TOKEN --batch-mode
```

Comment:
Reports to coveralls and then reports to Sonar.

Reply:
Do we want both or is the plan to disable coveralls reporting in a future step?

Reply:
I think we can retire coveralls when we're ready.  I'm up for trying this for a bit before making that decision.  I made a ticket to remind us.  [https://broadworkbench.atlassian.net/browse/DT-2130](https://broadworkbench.atlassian.net/browse/DT-2130)